packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -103,9 +104,9 @@ export const styles = (theme: ThemeType): JssStyles => ({
     alignItems: "baseline",
   },
   bottom: {
-    paddingBottom: 5,
-    fontSize: 12,
-    minHeight: 12
+    paddingBottom: isEAForum ? theme.spacing.unit : 5,
```

Comment:
Should this be using an "isFriendlyUI" thing? Are we doing that? We don't have to do that, it's pretty unimportant.

Reply:
I considered it, but I think it'll be cleaner to reserve that for switching components rather than working it into themes/styles (I've got some "friendly" checks in #6727). I think trying to make all 80 of these files backwards compatible for both LW _and_ the old EA forum design is going to make us pretty sad.